pkg/apis/minio.min.io/v1/types.go

```diff
@@ -51,6 +51,10 @@ type TenantScheduler struct {
 type TenantSpec struct {
 	// Definition for Cluster in given MinIO cluster
 	Zones []Zone `json:"zones"`
+	// Delete PVCs on tenant deletion. Defaults to (false) preserving PVCs if tenant is deleted. PVCs will
+	// need cleanup post tenant deletion if not set to true. If set to true ALL PVs for the tenant will be deleted.
+	// +optional
+	PurgePVCOnDeleteTenantCRD bool `json:"purgePVCOnDeleteTenantCRD,omitempty"`
```

Comment:
PurgePVCOnTenantDelete?

Reply:
or `PurgePVCOnDelete` perhaps, the whole CRD is about Tenant

Reply:
Renamed to be explicit that this is deleting Tenant